.codex/skills/commit-code/SKILL.md

```
@@ -0,0 +1,50 @@
+
+# Commit Code with HMIS Conventions
+
+Commit staged changes following the project's commit message conventions.
+
+## Arguments
+
+- `$0` - GitHub issue number (optional, for closing keyword)
+
+## Commit Message Format
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)